import-automation/executor/schedule_update_import.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function help {
+  echo "#Usage: -us <config_project_id> <absolute_import_path> <import_script_args>"
+  echo "## <config_project_id> is the GCP project ID where the config file is located." 
+  echo "## Update an import specified by <absolute_import_path>, e.g. scripts/us_usda/quickstats:UsdaAgSurvey"
+  echo "## Provide all args for the import script as one string, i.e. with quotes, e.g. \"--flag1=val1 --flag2=val2\""
+}
```

Comment:
`exit 1` to terminate if args are not correct?

Reply:
Done!